test/Yoti.Auth.Tests/ActivityTests.cs

```diff
@@ -10,50 +10,50 @@
-        private static readonly string _streetJson = "street";
-        private static readonly string _subdistrictJson = "subdistrict";
-        private static readonly string _districtJson = "district";
-        private static readonly string _postOfficeJson = "post_office";
-
-        private static readonly string _yotiAdminVerifierType = "YOTI_ADMIN";
-        private static readonly string _passportSourceType = "PASSPORT";
-        private static readonly string _drivingLicenseSourceType = "DRIVING_LICENCE";
-
-        private static readonly string _givenNamesAttribute = "given_names";
-        private static readonly string _familyNameAttribute = "family_name";
-        private static readonly string _fullNameAttribute = "full_name";
-        private static readonly string _genderAttribute = "gender";
-        private static readonly string _selfieAttribute = "selfie";
-        private static readonly string _phoneNumberAttribute = "phone_number";
-        private static readonly string _emailAddressAttribute = "email_address";
-        private static readonly string _nationalityAttribute = "nationality";
-        private static readonly string _ageOver18Attribute = "age_over:18";
-        private static readonly string _ageUnder18Attribute = "age_under:18";
-        private static readonly string _postalAddressAttribute = "postal_address";
-        private static readonly string _structuredPostalAddressAttribute = "structured_postal_address";
-        private static readonly string _dateOfBirthAttribute = "date_of_birth";
-
-        private static readonly DateTime _DateOfBirthValue = new DateTime(1980, 1, 13);
-        private static readonly string _dateOfBirthString = "1980-01-13";
+        private const string AddressFormatJson = "address_format";
+        private const string BuildingNumberJson = "building_number";
+        private const string AddressLineOneJson = "address_line1";
+        private const string TownCityJson = "town_city";
+        private const string PostalCodeJson = "postal_code";
+        private const string CountryIsoJson = "country_iso";
+        private const string CountryJson = "country";
+        private const string FormattedAddressJson = "formatted_address";
+        private const string StateJson = "state";
+        private const string CareOfJson = "care_of";
+        private const string BuildingJson = "building";
+        private const string StreetJson = "street";
+        private const string SubdistrictJson = "subdistrict";
+        private const string DistrictJson = "district";
+        private const string PostOfficeJson = "post_office";
+
+        private const string YotiAdminVerifierType = "YOTI_ADMIN";
+        private const string PassportSourceType = "PASSPORT";
+        private const string DrivingLicenseSourceType = "DRIVING_LICENCE";
+
+        private const string GivenNamesAttribute = "given_names";
+        private const string FamilyNameAttribute = "family_name";
```

Comment:
If we only use these once, we might as well just inline them

Reply:
We use GivenNames, Selfie, AgeOver18, AgeUnder18, PostalAddress, StructuredPostalAddress & DateOfBirth in multiple places. I thought it was quite neat having them all together, but do you think it would be better to inline the ones that aren't?

Reply:
Yes.  These tests are bulging.  Or if we really want them defined as constants, pull them out to another constants class - or maybe just use the constants defined in the SDK already